cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -1903,7 +1903,7 @@
                 },
                 "template": {
                   "type": "string",
-                  "description": "Inline template allowing users to define their\nown ``ntp_client`` configuration template.\nThe value must start with '## template:jinja'\nto enable use of templating support.\n"
+                  "description": "Inline template allowing users to customize their `ntp_client` configuration with the use of the Jinja templating engine. The template should start with '## template:jinja'. Within the template, you can utilize the following parameters:\n- servers: A list of strings, each specifying an NTP server. When defined, entries will be rendered as 'server [server_name] iburst'. Defaults to an empty list.\n- pools: A list of strings, each specifying an NTP pool. When defined, entries will be rendered as 'pool [pool_name] iburst'. Defaults to an empty list.\n- allow: A list of strings, each specifying a network or CIDR. When defined, entries will be rendered as 'allow [CIDR]'. Defaults to an empty list.\n- peers: A list of nodes that should peer with each other. When defined, entries will be rendered as 'peer [peer_name]'. Defaults to an empty list.\nFor more information on Jinja templates and syntax, refer to the official documentation: [Jinja Template Documentation](https://jinja.palletsprojects.com/en/2.9.x/templates/)"
```

Comment:
Thanks for making cloud-init better. Generally in review of module documentation changes, we like to run `tox -e doc; xdg-open doc/rtd_html/index.html and navigate to the specific doc file your are changing to see if the rendered style of variable-names and strings aligns with existing doc markdown/RST choices.

In your case, because of complexity of our doc generation from json schema, changing cloudinit/config/schemas/schema-cloud-config-v1.json gets rendered as part of the doc/rtd_html/reference/modules.html.

So to review your changes we look at:
```
tox -e doc 
xdg-open doc/rtd_html/reference/modules.html 
```
When I expand the schema tab on the cc_ntp module at /doc/rtd_html/reference/modules.html#ntp, I see a couple of suggestions I'd like to bring up to better render this content.

1. Let's avoid nested duplicated definitions or top-level keys supported by cc_ntp module. Having a large description here that repeats the descriptions from the other cc_ntp schema keys leads to an unreadable wall of text that may not help much and will lead to us having to update two places in the future if those keys change value or intent.
2.  We shoud be double-ticking around static strings and schema variable names instead of single tick to align with style throughout the docs. For example: \`\`## template: jinja\`\` instead of \`## template: jinja\` and \`\`servers\`\`: instead of servers:
3. Your '[markdown formatted link](url)' isn't being rendered in RST well. Let's drop that format in favor of '`Jinja Template Documentation <https://jinja.palletsprojects.com/en/2.9.x/templates/>`_'


 Maybe we can trim the description a bit to something like the following:
```
Inline template allowing users to customize their ``ntp_client`` configuration with the use of the Jinja templating engine. The template content should start with ``## template:jinja``. Within the template, you can utilize any of the following ntp module config keys: ``servers``, ``pools``, ``allow`` and ``peers``. Each cc_ntp schema config key is defined above. 
```

I made a couple of additional style changes as well to better decorate and render the RST in sphinx:
- let's double-tick around both variable names and static string values  \`\`## template: jinja\`\` instead of \`## template: jinja\`   and also \`\`servers\`\`: instead of no decoration around servers:
- 


It may be worth a separate minor doc pull request to add those RST style suggestions for schema variable names and/or static string RST decorators to doc/rtd/development/docs.rst